common/src/main/resources/startup.properties

```diff
@@ -243,6 +243,9 @@ it.workflow.execution.listeners=org.apache.falcon.catalog.CatalogPartitionHandle
 # Authorization Enabled flag: false (default)|true
 *.falcon.security.authorization.enabled=false
 
+# CSRF filter enabled flag: false (default) | true
+*.falcon.security.csrf.enabled=false
```

Comment:
The first question is if we have dependency on authorization options? The current filter actually works with both simple and kerberos authentication we provided in Falcon. See more comments above.